pact.cabal

```diff
@@ -82,6 +87,10 @@ library
                      , Pact.Types.Typecheck
                      , Pact.Typechecker
 
+  if flag(ghc-flags)
```

Comment:
What's all this?

Reply:
I use this flag i `cabal.project.local` to get `.ghc.flags` and `.ghc.version` which gets me some better tooling data. It never touches prod code.

Reply:
kk